spec/controllers/collections_controller_spec.rb

```diff
@@ -72,7 +72,10 @@ def to_solr(solr_doc={})
       @asset1 = ActiveFedora::Base.create!
       @asset2 = ActiveFedora::Base.create!
       post :create, batch_document_ids: [@asset1, @asset2], collection: {title: "My Secong Collection ", description: "The Description\r\n\r\nand more"}
-      expect(assigns[:collection].members).to eq([@asset1, @asset2])
+      members = assigns[:collection].members
+      expect(members).to include @asset1
+      expect(members).to include @asset2
+      expect(members.size).to eq 2
```

Comment:
You should be able to replace lines 75-78 by using a more specific matcher:

```ruby
expect(assigns[:collection].members).to contain_exactly(@asset1, @asset2)
```

`contain_exactly` shouldn't care about order.